Cue/app/api/Library.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import { serverURL } from '../env';
+
+
+var LibraryApi = {
+	//TODO: not sure proper way to get state in the api
```

Comment:
That sounds like a good plan. Ill make an issue for this and assign to myself, but don't think we need this change for the demo.